### PR TITLE
[GraphQL] Update `graphql` schema

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Build failures; lint failures

File: server/internal/graphql/model/helpers.go [hunk 1]
```diff
@@ -345,6 +345,18 @@ func GetInternalControllerStatus(status controllers.MesheryControllerStatus) Mes
 
 	case controllers.Unknown:
 		return MesheryControllerStatusUnkown
+
+	case controllers.Undeployed:
```
Comment (leecalcote) [inline server/internal/graphql/model/helpers.go:349]: Please link to where these statuses are documented in Meshery Docs.

Comment (leecalcote) [reply server/internal/graphql/model/helpers.go:349]: If they aren't, please document and include in this PR.

**Comment by MUzairS15** [comment]
> Build failures; lint failures

There aren't any build failures.

Lint failures addressed.

**Comment by MUzairS15** [comment]
Failing GraphQL validation is because of outdated workflow I assume.

**Comment by leecalcote** [comment]
@MUzairS15 the UI is still failing to build.

**Comment by leecalcote** [comment]
@MUzairS15 merge conflicts.

**Pull request merged**